RealmLoginKit Android/.idea/copyright/profiles_settings.xml

```diff
@@ -0,0 +1,3 @@
+<component name="CopyrightManager">
```

Comment:
Do we need to include some files in `.idea/` directory?
If there is no particular reason, we should use the same `.gitignore` with other demo apps or `raelm-java`